app/src/pages/inside/stepPage/modals/makeDecisionModal/elements/itemHeader/itemHeader.jsx

```diff
@@ -35,50 +44,83 @@ export const ItemHeader = ({
     id,
     name,
     issue: { autoAnalyzed, ignoreAnalyzer, issueType, externalSystemIssues },
     patternTemplates,
   } = item;
+  const defectTypes = useSelector(defectTypesSelector);
   const getLogItemLink = useSelector(getLogItemLinkSelector);
   const link = getLogItemLink(item);
 
   return (
     <div
-      className={cx('item-info', { selected: isSelected, preselected })}
-      onClick={() => selectItem(id)}
+      className={cx('item-info', {
+        height40: mode === SIMILAR_TO_INVESTIGATE_ITEM,
+      })}
+      onClick={() => toggleDetails()}
     >
-      <div className={cx('header', { 'narrow-view': isNarrowView })}>
-        <Link
-          to={link}
-          target="_blank"
-          className={cx('item-name', { 'narrow-view': isNarrowView })}
-          onClick={onClickLinkEvent}
-        >
-          <span title={name}>{name}</span>
-          <div className={cx('icon')}>{Parser(ExternalLinkIcon)}</div>
-        </Link>
-        <div className={cx('defect-block', { 'narrow-view': isNarrowView })}>
-          {!hideLabels && ignoreAnalyzer && <IgnoredInAALabel className={cx('ignore-aa-label')} />}
-          {!hideLabels && autoAnalyzed && <AutoAnalyzedLabel className={cx('aa-label')} />}
-          {!hideLabels && !!patternTemplates.length && (
-            <PatternAnalyzedLabel
-              patternTemplates={patternTemplates}
-              className={cx('pa-label')}
-              showTooltip
-            />
+      <div className={cx('header')}>
+        {mode !== ANALYZED_ITEM && (
+          <div>
+            {mode === SIMILAR_TO_INVESTIGATE_ITEM && (
+              <InputCheckbox
+                className={cx('checkbox-margin-right')}
+                value={isSelected}
+                onChange={() => selectItem(id)}
+              />
+            )}
+            {mode === HISTORY_LINE_ITEM && (
+              <InputRadio
+                small
+                mode={'dark'}
+                value={isSelected}
+                ownValue
+                onChange={() => selectItem(id)}
+              />
+            )}
+          </div>
+        )}
+        <div className={cx('header-content')}>
+          <Link to={link} target="_blank" className={cx('item-name')} onClick={onClickLinkEvent}>
+            <span title={name}>{name}</span>
+            <div className={cx('icon')}>{Parser(ExternalLinkIcon)}</div>
+          </Link>
+          {mode === ANALYZED_ITEM && (
+            <div className={cx('defect-block')}>
+              {!hideLabels && ignoreAnalyzer && (
+                <IgnoredInAALabel className={cx('ignore-aa-label')} />
+              )}
+              {!hideLabels && autoAnalyzed && <AutoAnalyzedLabel className={cx('aa-label')} />}
+              {!hideLabels && !!patternTemplates.length && (
+                <PatternAnalyzedLabel
+                  patternTemplates={patternTemplates}
+                  className={cx('pa-label')}
+                  showTooltip
+                />
```

Comment:
Should we show PA, AA, ignore AA labels only for analyzed item or for all items if labels are exist?